R/ancova.R

```diff
@@ -12,6 +12,8 @@
 #' fit the ancova model at. If `NULL`, a separate ancova model will be fit to the
 #' outcomes from each visits (as determined by `unique(data[[vars$visit]])`).
 #' See details.
+#' @param weights Character, specifies wether to use "proportional" or "equal" weighting for each
+#' categorical covariate combination when calculating the lsmeans.
```

Comment:
Suggest to add the following to the help: 
"proportional" is the default and is equivalent to standardization, i.e. lsmeans in each treatment group are equal to the predicted mean outcome from the ancova model for that treatment group based on baseline characteristics of all subjects regardless of their assigned treatment.

Reply:
@nociale  & @wolbersm ,  how about the following:

```
@param weights Character, either "proportional" (default) or "equal". Specifies the
weighting strategy to be used for categorical covariates when calculating the lsmeans.
See details.
```

and then:

```
 Weighting:

       "proportional" is the default scheme that is used. This is
       equivalent to standardization, i.e. the lsmeans in each group
       are equal to the predicted mean outcome from the ancova model
       for that group based on baseline characteristics of all subjects
       regardless of their assigned group. The alternative weighting
       scheme, "equal", creates hypothetical patients by expanding out
       all combinations of the models categorical covariates. The
       lsmean is then calculated as the average of the predicted mean
       outcome for these patients assuming they come from each of the
       groups in turn.

       In short:

         • "equal" weights categorical covariates equally across all
           theoretical combinations

         • "proportional" weights categorical covariates based upon
           their frequency of occourance in the data

```

(added in my most recent commit)

Reply:
Thanks @gowerc. Looks good to me except for "occourance" -> "occurrence".